subprojects/docs/src/docs/userguide/dependencyLocking.adoc

```diff
@@ -51,14 +61,14 @@ Or the following, as a way to lock all configurations:
 
 [NOTE]
 ====
-Only configurations that can be resolved will have a lock file attached to them. Applying locking on non resolvable-configurations is simply a no-op.
+Only configurations that can be resolved will have lock state attached to them. Applying locking on non resolvable-configurations is simply a no-op.
 ====
 
-=== Generating and updating lockfiles
+=== Generating and updating lock state
```

Comment:
Maybe "dependency locks" would sound better from an external point of view. No strong opinion.

Reply:
Good point, changing